.github/workflows/site.yaml

```diff
@@ -40,9 +40,11 @@ jobs:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0
-      - uses: astral-sh/setup-uv@v3
+      - uses: astral-sh/setup-uv@v6
       - run: uv python install
       - uses: actions/setup-node@v4
+        with:
+          node-version: 18
```

Comment:
Is it possible for this to be more modern? IIUC 18 is EOL (as of yesterday, but still)

Reply:
Oh, is it? I need to get with the times. Updating to 22.